src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -77,16 +114,20 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
   }
 
   when(CallRunningAbortUnavailable) {
-    case Event(RegisterCallAbortFunction(abortFunction), _) => goto(CallRunningAbortAvailable) using Option(abortFunction)
+    case Event(RegisterCallAbortFunction(newAbortFunction), data) =>
+      val updatedData = data.copy(abortFunction = Option(newAbortFunction))
+      goto(CallRunningAbortAvailable) using updatedData
     case Event(AbortCall, _) => goto(CallRunningAbortRequested)
```

Comment:
I didn't intentionally make changes to the abort logic, but it looks like what happens here is that if an abort is requested but there's no abort available, there's a transition to `CallRunningAbortRequested` but no actual attempt is made to abort.  If subsequently an abort function is registered, then the call will `tryAbort`.  Sound right @cjllanwarne?